pkg/pillar/rootfs/init.sh

```diff
@@ -7,6 +7,7 @@ echo 'nameserver 8.8.8.8' > /etc/resolv.conf
 
 #Copy pre-defined fscrypt.conf
 cp fscrypt.conf /etc/fscrypt.conf
+cp periodic_sigusr1.sh /sbin/
```

Comment:
If this script is a one-off (e.g. only useful in the context of a fully-assembled pillar running as part of EVE) why wouldn't you put this file directly into pkg/pillar/rootfs/sbin/periodic_sigusr1.sh  ? That will avoid this copy and make it more consistent with the rest of of what we do.

If this sctipt is useful for somebody developing EVE on their laptop -- you should put it under scripts/